apps/src/music/player/MusicPlayer.ts

```diff
@@ -560,7 +563,7 @@ export default class MusicPlayer {
   }
 
   private calculatePitchShift(soundData: SoundData) {
```

Comment:
I believe this function is only used for pack sounds (not instruments/kits) - in which case, could change the signature to make the input parameter a `SoundDataPack` and then you wouldn't need to check for `type` below. Might also help clarify what types of sounds are actually affected by this